logging/jul/src/main/java/io/helidon/logging/jul/JulMdcPropagator.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2020 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.logging.jul;
+
+import java.util.Map;
+
+import io.helidon.common.context.spi.DataPropagationProvider;
+
+/**
+ * This is propagator of JUL MDC values between different threads.
```

Comment:
Javadoc should mark it as a service loader service, so people do not try to use it directly